bin/lib/autovoice.js

```diff
@@ -1,145 +1,102 @@
-const fetch = require('node-fetch');
-const md5   = require('md5');
-const debug = require('debug')('autovoice:lib');
+const   md5 = require('md5');
+const debug = require('debug')('bin:lib:autovoice');
 
 const             extractUuid = require('./extract-uuid');
 const            parseRSSFeed = require('./parse-rss-feed');
 const                     tts = require('./get-tts');
 const          dataItemsCache = require('./dataItemsCache');
 const            constructRSS = require('./constructRSS');
 const formatContentForReading = require('./formatContentForReading');
+const            fetchContent = require('./fetchContent');
```

Comment:
the word 'fetch' indicates that fundamentally a fetch promise is involved (and returned), and 'Content' hints at a variety of FT-y data which requires extra handling.

fetchAssortedContent, fetchAndMungeContent ?